src/PlugKeyRing/modules/NetworkModule/Network.ts

```diff
@@ -98,6 +98,11 @@ export class Network {
 
   public registerToken = async ({ canisterId, standard, walletId, secretKey, logo }: { canisterId: string, standard: string, walletId: number, secretKey: BinaryBlob, logo?: string }) => {
     const token = this.registeredTokens.find(({ canisterId: id }) => id === canisterId);
+    const defaultToken = this.defaultTokens.find(({ canisterId: id }) => id === canisterId);
+    if (defaultToken) {
+      await this.onChange?.();
+      return this.defaultTokens;
```

Comment:
shouldn't we return the registeredTokens list here?

Reply:
Nope if token is into defaultToken, we not gonna use registeredTokens